tests/models/test_llava.py

```diff
@@ -65,9 +66,15 @@ def vllm_to_hf_output(vllm_output: Tuple[List[int], str],
     return hf_input_ids, hf_output_str
 
 
+# TODO: remove this after CPU float16 support ready
+target_dtype = "float"
+if torch.cuda.is_available():
+    target_dtype = "half"
```

Comment:
I found the previous thread lock is caused by the `half` dtype as well. (maybe because of the dtype casting)

How about using `bfloat16` for CPU here? It seems that `bfloat16` also works on Xeon CPU in transformers.
```python
target_dtype = "half"
if is_cpu():
    target_dtype = "bfloat16"
```